dhall-docs/src/Dhall/Docs.hs

```diff
@@ -144,66 +147,82 @@ saveHtml
                                 --   Used to remove the prefix from all other dhall
                                 --   files in the package
     -> Path Abs Dir             -- ^ Output directory
+    -> String                   -- ^ Package name
     -> (Path Abs File, Header)  -- ^ (Input file, Parsed header)
     -> IO (Path Abs File)       -- ^ Output path file
-saveHtml inputAbsDir outputAbsDir t@(absFile, _) = do
-    htmlOutputFile <- (outputAbsDir </>)
-            <$> (Path.stripProperPrefix inputAbsDir absFile
-                >>= addHtmlExt)
+saveHtml inputAbsDir outputAbsDir packageName t@(absFile, _) = do
+    htmlOutputFile <- do
+        strippedPath <- Path.stripProperPrefix inputAbsDir absFile
+        strippedPathWithExt <- addHtmlExt strippedPath
+        return (outputAbsDir </> strippedPathWithExt)
 
     let htmlOutputDir = Path.parent htmlOutputFile
 
     Path.IO.ensureDir htmlOutputDir
 
-    let relativeResources = resolveRelativePath outputAbsDir htmlOutputDir
+    let relativeResourcesPath = resolveRelativePath outputAbsDir htmlOutputDir
 
     Lucid.renderToFile (Path.fromAbsFile htmlOutputFile)
-        $ filePathHeaderToHtml t (relativeResources <> "index.css")
+        $ filePathHeaderToHtml t DocParams {..}
+
     return htmlOutputFile
   where
     addHtmlExt :: Path b File -> IO (Path b File)
-#if MIN_VERSION_path(0,7,0)
     addHtmlExt = Path.addExtension ".html"
-#else
-    addHtmlExt = Path.addFileExtension "html"
-#endif
 
 {-| Create an index.html file on each folder available in the second argument
     that lists all the contents on that folder.
 
-    Examples:
+    For example,
 
-    >>> createIndexes $(mkAbsDir "/") [
-        , $(mkAbsFile "/a/b.txt")
-        , $(mkAbsFile "/a/c/b.txt")
-        , $(mkAbsFile "/a/c.txt")
+    @
+    createIndexes [absdir|/|]
+        [ [absfile|/a/b.txt|]
+        , [absfile|/a/c/b.txt|]
+        , [absfile|/a/c.txt"|]
         ]
+    @
 
     ... will create two index.html files:
 
-    1. $(mkAbsFile "/a/index.html"), that will list the @"/a/b.txt"@ and
-    @"/a/c.txt"@ files
-    2. $(mkAbsFile "/a/c/index.html") that will list the @"a/c/b.txt"@ file
+    1. @/a/index.html@, that will list the @/a/b.txt@ and
```

Comment:
Check how this documentation renders.  The reason I ask is that you might need to escape the forward slashes, because I think haddock treats them as italics, even inside of `@`